site/content/introduction/01-getting-started.md

```diff
@@ -25,7 +25,7 @@ eksctl create cluster --name=cluster-1 --nodes=4
 
 ```
 
-EKS supports versions `1.12`, `1.13`, `1.14` (default) and `1.15`.
+EKS supports versions `1.12`, `1.13`, `1.14` and `1.15` (default).
 With `eksctl` you can deploy either version by passing `--version`.
```

Comment:
I think we should also change this text since we have more than 2 versions to select from.

Something along these lines:
```suggestion
With `eksctl` you can deploy any of the supported versions by passing `--version`.
```